content/c-sharp/concepts/math-functions/terms/acos/acos.md

```diff
@@ -65,3 +65,22 @@ NaN
 NaN
 1.5707963267949
 ```
+
+## Codebyte Example
+
+We use the `Math.Acos` function to calculate the arc cosine of `x`. The result is stored in the `result` variable.
```

Comment:
```suggestion
The example below uses `Math.Acos()`  to calculate the arc cosine of `x`. The result is stored in the `result` variable.
```

Reply:
In docs we generally try to avoid the use of first and second person pronouns (I, you, we)

Reply:
ah very good tip thanks! 📝